1-callbacks/solution.js

```diff
@@ -33,15 +33,41 @@ node solution.js name1 name2 name3
 */
 
 function solution() {
-    // YOUR SOLUTION GOES HERE
-
-    // you get your 5 names here
-
-    // iterate the names array and validate them with the method
-
-    // log the final result
+  // YOUR SOLUTION GOES HERE
+
+  // you get your 5 names here
+
+  // Using an array:
+  // const users = ["Mary", "Daniel", "Stacy", "Gerardo", "Diego"];
+  
+  // Using process.argv property:
+  const { argv } = require('process');
+  const users = argv.filter((item, index) => index > 1);
+
+  // iterate the names array and validate them with the method
+  const validateUser = require("./validate-user");
+  const successList = [],
+  failureList = [];
+  for (const user of users) {
+    validateUser(user, (error, response) => {
+      if (error) {
+        failureList.push(error.message);
+      } else {
+        successList.push(response);
+      }
```

Comment:
Jus a suggestion, since our output should be a string, we could  change 'failureList'. and  'successList' as strings and add the new values (+). In this way we could omit those 2 loops.